src/Interactive/Host/Microsoft.CodeAnalysis.InteractiveHost.csproj

```diff
@@ -57,6 +57,10 @@
     <Compile Include="..\..\Compilers\Core\Portable\FileSystem\PathUtilities.cs" Link="Utilities\PathUtilities.cs" />
     <Compile Include="..\..\Compilers\Core\Portable\FileSystem\PathKind.cs" Link="Utilities\PathKind.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="StreamJsonRpc" Version="2.2.34" />
```

Comment:
>2.2.34 [](start = 55, length = 6)

We have msbuild variables defined for the versions. This would be `Version="$(StreamJsonRpcVersion)"`.
You shouldn't need to list Newtonsoft.Json package directly as it will be pulled in by StreamJsonRpc.